internal/transactions/sign.go

```diff
@@ -54,22 +56,43 @@ func sign(
 	services *services.Services,
 	state *flowkit.State,
 ) (command.Result, error) {
-	filename := args[0]
-	payload, err := readerWriter.ReadFile(filename)
+	var payload []byte
+	var err error
+	yesOption := globalFlags.Yes
+	filenameOrUrl := args[0]
+
+	if signFlags.FromRemoteUrl {
+		payload, err = services.Transactions.GetRlp(filenameOrUrl)
+		yesOption = false // force user to view remote tx
+	} else {
+		payload, err = readerWriter.ReadFile(filenameOrUrl)
+	}
+
 	if err != nil {
-		return nil, fmt.Errorf("failed to read partial transaction from %s: %v", filename, err)
+		return nil, fmt.Errorf("failed to read partial transaction from %s: %v", filenameOrUrl, err)
 	}
 
 	signer, err := state.Accounts().ByName(signFlags.Signer)
 	if err != nil {
 		return nil, fmt.Errorf("signer account: [%s] doesn't exists in configuration", signFlags.Signer)
 	}
 
-	signed, err := services.Transactions.Sign(signer, payload, globalFlags.Yes)
+	signed, err := services.Transactions.Sign(signer, payload, yesOption)
 	if err != nil {
 		return nil, err
 	}
 
+	if signFlags.FromRemoteUrl {
+		tx := signed.FlowTransaction()
+		signedRlp := fmt.Sprintf("%x", string(tx.Encode()))
```

Comment:
is this intended to stay in the final command? if so we may want to add more context?